package.json

```diff
@@ -22,7 +22,7 @@
     "watch:cssjs": "nodemon --ext css,js --ignore tests --ignore public/assets/css/critical.css --ignore public/assets/js/main.dev.js --ignore public/assets/js/bundle.dev.js --exec \"npm run build:dev\"",
     "watch:other": "nodemon app.js --ext pug,svg,png,yml --ignore tests",
     "start-node": "node app.js",
-    "start": "npm run build:prod && cross-env NODE_ENV=production forever start --plain app.js",
+    "start": "npm run build:prod && cross-env NODE_ENV=production && node app.js",
```

Comment:
1. Won't this break `npm stop` and `npm restart` scripts?
2. You don't need `&&` BTW
3. This won't run `build:prod` which is needed for the files to be generated...